src/ggml-cpu/ops.cpp

```diff
@@ -6064,6 +6064,178 @@ void ggml_compute_forward_conv_transpose_2d(
     }
 }
 
+// ggml_compute_forward_depthwise_conv_2d
+
+struct ggml_depthwise_conv_2d_params {
```

Comment:
I think it's preferable to also rename the symbols throughout - although not really necessary as these are temporary, it's mainly to keep consistency and ease fuzzy searches:

- `ggml_depthwise_conv_2d_params` -> `ggml_conv_2d_dw_params`
- `GGML_OP_DEPTHWISE_CONV_2D` -> `GGML_OP_CONV_2D_DEPTHWISE` or `GGML_OP_CONV_2D_DW`
- `ggml_compute_forward_depthwise_conv_2d` -> `ggml_compute_forward_conv_2d_dw`
- `test-depthwise-conv2d.cpp` -> `test-conv2d-dw.cpp`
- etc.

Reply:
Done - I liked searching for "depthwise", but yes consistency and finding all places with one search is nice.